install/0000_80_machine-config-operator_09_etcddefragmentation_cronjob.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: etcd-defrag
+  namespace: openshift-etcd
+spec:
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          hostNetwork: true
+          containers:
+            - command: ["/usr/bin/sh"]
+              terminationMessagePolicy: FallbackToLogsOnError
+              args:
+                - "-c"
+                - |
+                  set -euo pipefail
+
+                  export ETCDCTL_API=3
+                  export ETCDCTL_CACERT=/etc/ssl/etcd/ca.crt ETCDCTL_CERT=$(find /etc/ssl/ -name *peer*crt) ETCDCTL_KEY=$(find /etc/ssl/ -name *peer*key)
+                  export ETCDCTL_ENDPOINTS=https://127.0.0.1:2379;
+                  for m in $(etcdctl member list); do
```

Comment:
let's hardcode the endpoint vs setting ENV. in 3.4 you can't have both ENV and flag so we should do this here nd be explicit.

Reply:
I guess we should probably do localhost vs 127.0.0.1 in case of IPv6